settings.py

```diff
@@ -496,6 +496,23 @@ def validate(cls, path: str) -> None:
                 raise ValueError(f"File hash does not match for {path}")
 
 
+class NESRomPath(UserFilePath):
+    """Special UserFilePath that ignores a non-optional variable header when validating. Using this will require
+    validating that the rom has a header when loading the rom, and adding one should it be missing."""
+
+    @classmethod
+    def validate(cls, path: str) -> None:
+        with open(path, "rb", buffering=0) as f:
```

Comment:
This method would benefit from a few comments, like the one in SNESRomPath above. Like, what is the 16, etc.
Maybe a few variables could have a better name too (`f`?), but this one has precedent in the rest of the file so I guess we can live with it.

Reply:
I agree that a comment where this seeks would be great.

`f` is expected and understandable for a single file steam.